swagger.yaml

```diff
@@ -6046,6 +6046,53 @@ paths:
             $ref: '#/definitions/error'
 
 definitions:
+  ovirt-platform:
+    type: object
+    description: oVirt platform-specific configuration upon which to perform the installation.
+    properties:
+      fqdn:
+        type: string
+        description: The oVirt's engine fully qualified domain name.
+        x-nullable: true
+      username:
+        type: string
+        description: The user name to use to connect to the oVirt instance.
+        x-nullable: true
+      password:
```

Comment:
In the past, we tried not to return user passwords from the service.  For example, once pull secret is specified via the API, it cannot be retrieved - we have a boolean pull_secret_set to indicate that some value was provided.  I realize that we have the same issue with the vSphere API - I just opened a task to fix that.

Reply:
Would it work for you if I workaround the issue in this PR by overwriting the ovirt password with `nil` just before the API response is delivered? If you agree, I will add a FIXME and open a bug to do the final implementation.

Reply:
That would be confusing to users because they set it and it disappears.

Reply:
I'm OK with handling this in a separate PR